mmdet/datasets/samplers/infinite_sampler.py

```diff
@@ -56,7 +57,8 @@ def __init__(self,
         # in the same order based on the same seed. Then different ranks
         # could use different indices to select non-overlapped data from the
         # same data list.
-        self.seed = sync_random_seed(seed)
+        device = get_device()
```

Comment:
The `InfiniteBatchSampler` in this file has the same problem. Would you like to fix it in this PR?

Reply:
Ok I have fixed it.